libs/fgdbg/client/build.gradle.kts

```diff
@@ -27,6 +28,7 @@ dependencies {
         implementation("io.ktor:ktor-client-core:$ktorVersion")
         implementation("io.ktor:ktor-client-cio:$ktorVersion")
         implementation("io.ktor:ktor-client-websockets:$ktorVersion")
+        implementation("com.charleskorn.kaml:kaml:0.51.0")
```

Comment:
Adding a third party dep to parse YAML, is this OK?

Reply:
@romainguy what do you think?

Reply:
It's Apache 2, fine with me